tools/system_libs.py

```diff
@@ -530,6 +530,32 @@ def get_default_variation(cls, **kwargs):
     return super(MTLibrary, cls).get_default_variation(is_mt=shared.Settings.USE_PTHREADS, **kwargs)
 
 
+class OptimizedAggressivelyForSizeLibrary(Library):
```

Comment:
Can we come up with more better name?   How about just `SizeOptimizedLibrary`?

Reply:
I think for the same reason as above, that would not be as good. Maybe `OptimizedForOzLibrary` if something? I want the name reflect via static reading that it is for `-Oz` and not for `-Os`